posts/2023-02-20-liberty-tools-eclipse-deep-dive.adoc

```diff
@@ -0,0 +1,118 @@
+seo-title: Upgrade your cloud-native Java development experience with Liberty Tools for Eclipse IDE
+seo-description: Take a deep dive into the features of the latest early release of Liberty Tools for Eclipe IDE
+blog_description: "Take a deep dive into the features of the latest early release of Liberty Tools for Eclipe IDE"
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-actions.png
+---
+= Upgrade your cloud-native Java development experience with Liberty Tools for Eclipse IDE
+Adam Wisniewski <https://github.com/awisniew90>
+
+Liberty Tools for Eclipse IDE provides a simplified yet powerful development experience. With capabilities like dev mode, Liberty config editing, and Jakarta EE and MicroProfile code assistance, you can code and manage your application without having to leave your IDE.
+
+Try it out today by installing from the link:https://marketplace.eclipse.org/content/liberty-tools[Eclipse Marketplace].
+
+Check out the following video for a deep dive walkthrough:
+
+++++
+<iframe width="560" height="315" align="center" src="https://www.youtube.com/watch?v=OcgR_4BL12o" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+Let's dive in!
+
+== What is Liberty Tools?
+
+Liberty Tools is the next generation IDE tooling for developing cloud-native Java applications with Liberty. It takes an app-centric approach, leveraging the capabilities of the Liberty Maven and Liberty Gradle plugins, so you no longer have to spend time creating and managing Liberty instances. By running dev mode behind the scenes, the Liberty runtime is automatically downloaded and started, and your app is built, packaged and deployed to it. To save even more time, listeners detect any changes you make, and your app is automatically rebuilt and redeployed without having to restart the runtime.
+
+Keeping in line with the app-centric design, Liberty Tools provides code assistance for both Jakarta EE and MicroProfile APIs. Code completion, hover descriptions, diagnostics and quick-fixes save time, reduce errors, and keep you focused on your code. Liberty Tools accomplishes this by integrating with the link:https://github.com/eclipse/lsp4jakarta[Language Server for Jakarta EE] and the link:https://github.com/eclipse/lsp4mp[Language Server for MicroProfile], both open source projects which are continually adding new features and support.
+
+While much of the runtime mechanics are automated with Liberty Tools, some runtime configuration is, of course, necessary. However, this process has been streamlined with config assistance for `server.xml`, `bootstrap.properties`, and `server.env` files. Rather than having to refer to the documentation to find the correct Liberty feature to add or the appropriate value to set for your specific logging attribute, Liberty Tools provides lists of options and descriptions all without having to leave Eclipse.
```

Comment:
to set for your specific logging attribute
  =>
to set for your specific logging attribute, (for example),...

---
Maybe??  Getting right into logging seems maybe overly specific.. I get what you're saying but perhaps clearer adding that?